README.md

```diff
@@ -6,48 +6,48 @@ goldmark
 [![https://goreportcard.com/report/github.com/yuin/goldmark](https://goreportcard.com/badge/github.com/yuin/goldmark)](https://goreportcard.com/report/github.com/yuin/goldmark)
 
-> A markdown parser written in Go. Easy to extend, standard compliant, well structured.
+> A Markdown parser written in Go. Easy to extend, standard compliant, well structured.
 
-goldmark is compliant to CommonMark 0.29.
+goldmark is compliant with CommonMark 0.29.
 
 Motivation
 ----------------------
-I need a markdown parser for Go that meets following conditions:
+I need a Markdown parser for Go that meets following conditions:
 
 - Easy to extend.
-    - Markdown is poor in document expressions compared with other light markup languages like restructuredText.
-    - We have extended a markdown syntax. i.e. : PHPMarkdownExtra, Github Flavored Markdown.
+    - Markdown is poor in document expressions compared with other light markup languages like reStructuredText.
+    - We have extensions to the Markdown syntax, e.g. PHP Markdown Extra, GitHub Flavored Markdown.
 - Standard compliant.
     - Markdown has many dialects.
-    - Github Flavored Markdown is widely used and it is based on CommonMark aside from whether CommonMark is good specification or not.
+    - GitHub Flavored Markdown is widely used and it is based on CommonMark aside from whether CommonMark is good specification or not.
         - CommonMark is too complicated and hard to implement.
 - Well structured.
     - AST based, and preserves source position of nodes.
 - Written in pure Go.
 
-[golang-commonmark](https://gitlab.com/golang-commonmark/markdown) may be a good choice, but it seems copy of the [markdown-it](https://github.com/markdown-it) .
+[golang-commonmark](https://gitlab.com/golang-commonmark/markdown) may be a good choice, but it seems to be a copy of the [markdown-it](https://github.com/markdown-it).
 
-[blackfriday.v2](https://github.com/russross/blackfriday/tree/v2) is a fast and widely used implementation, but it is not CommonMark compliant and can not be extended from outside of the package since it's AST is not interfaces but structs. 
+[blackfriday.v2](https://github.com/russross/blackfriday/tree/v2) is a fast and widely used implementation, but it is not CommonMark compliant and cannot be extended from outside of the package since its AST uses not interfaces but structs.
 
-Furthermore, its behavior differs with other implementations in some cases especially of lists.  ([Deep nested lists don't output correctly #329](https://github.com/russross/blackfriday/issues/329), [List block cannot have a second line #244](https://github.com/russross/blackfriday/issues/244), etc).
+Furthermore, its behavior differs from other implementations in some cases especially of lists.  ([Deep nested lists don't output correctly #329](https://github.com/russross/blackfriday/issues/329), [List block cannot have a second line #244](https://github.com/russross/blackfriday/issues/244), etc).
```

Comment:
"in some cases, especially regarding lists", maybe? The comma is necessary, but I'm not sure about the rewording